### PR TITLE
Error setting CLOB parameter for stored procedure

### PR DESCRIPTION
**Pull request by tnleeuw**

### CODE REVIEW

**Comment by alisihab** [comment]
jdbcutil changes seems to work

File: core/src/main/java/nl/nn/adapterframework/jdbc/StoredProcedureQuerySender.java [hunk 1]
```diff
@@ -62,7 +62,9 @@
  *     have a corresponding {@link Parameter} entry. Output parameters should have {@code mode="OUTPUT"}, or
  *     {@code mode="INOUT"} depending on how the stored procedure is defined.
  * </p>
- * <p><b>NOTE:</b> See {@link DB2XMLWriter} for ResultSet!</p>
+ * <p><em>NOTE 1:</em> Support for stored procedures is currently experimental and changes in the output-format procuded
```
Comment (jkosternl) [inline core/src/main/java/nl/nn/adapterframework/jdbc/StoredProcedureQuerySender.java:65]:
```suggestion
 * <p><em>NOTE 1:</em> Support for stored procedures is currently experimental and changes in the currently produced output-format
```

File: core/src/test/java/nl/nn/adapterframework/jdbc/StoredProcedureQuerySenderTest.java [hunk 2]
```diff
@@ -203,7 +204,42 @@ public void testSimpleStoredProcedureClobInputParameter() throws Exception {
 		// Assert
 		assertTrue(result.isSuccess());
 		String clobValue = getClobValueAsString(id);
-		assertEquals("test-data", clobValue);
+		assertEquals(TEST_DATA_STRING, clobValue);
+	}
+	@Test
```
Comment (jkosternl) [inline core/src/test/java/nl/nn/adapterframework/jdbc/StoredProcedureQuerySenderTest.java:209]:
```suggestion

	@Test
```

File: core/src/main/java/nl/nn/adapterframework/jdbc/StoredProcedureQuerySender.java [hunk 3]
```diff
@@ -154,7 +156,6 @@ protected PreparedStatement prepareQueryWithResultSet(Connection con, String que
 			} else {
 				typeNr = JdbcUtil.mapParameterTypeToSqlType(param.getType()).getVendorTypeNumber();
 			}
-			callableStatement.setNull(position, typeNr);
```
Comment (jkosternl) [inline core/src/main/java/nl/nn/adapterframework/jdbc/StoredProcedureQuerySender.java:157]: Should we apply this fix also on 7.8? Lot of projects are still migrating to 7.8, as what I understood. Or is that risky?

**Review by jkosternl — APPROVED** [review]

**Review by nielsm5 — APPROVED** [review]

File: core/src/main/java/nl/nn/adapterframework/jdbc/StoredProcedureQuerySender.java (hunk 3 above)
Comment (tnleeuw) [reply core/src/main/java/nl/nn/adapterframework/jdbc/StoredProcedureQuerySender.java:157]: 7.9 only, not 7.8. It does not seem to harm in tests. Only affects those who use stored procedures, which is very few now.

Comment (jkosternl) [reply core/src/main/java/nl/nn/adapterframework/jdbc/StoredProcedureQuerySender.java:157]: Ok, great, thanks!

**Pull request merged**